Index: configure.ac
===================================================================
--- configure.ac	(revision 45655)
+++ configure.ac	(working copy)
@@ -2157,6 +2157,7 @@
   packaging/svr4/pkginfo
   plugins/Makefile
   plugins/asn1/Makefile
+  plugins/ceph/Makefile
   plugins/docsis/Makefile
   plugins/ethercat/Makefile
   plugins/gryphon/Makefile
Index: Makefile.am
===================================================================
--- Makefile.am	(revision 45655)
+++ Makefile.am	(working copy)
@@ -277,6 +277,7 @@
 -include plugins/Custom.make
 plugin_ldadd = $(_CUSTOM_plugin_ldadd_) \
 	-dlopen plugins/asn1/asn1.la \
+	-dlopen plugins/ceph/ceph.la \
 	-dlopen plugins/docsis/docsis.la \
 	-dlopen plugins/ethercat/ethercat.la \
 	-dlopen plugins/gryphon/gryphon.la \
Index: plugins/Makefile.nmake
===================================================================
--- plugins/Makefile.nmake	(revision 45655)
+++ plugins/Makefile.nmake	(working copy)
@@ -8,6 +8,7 @@
 
 PLUGIN_LIST = \
 	asn1        \
+	ceph        \
 	docsis      \
 	ethercat    \
 	gryphon     \
Index: plugins/Makefile.am
===================================================================
--- plugins/Makefile.am	(revision 45655)
+++ plugins/Makefile.am	(working copy)
@@ -24,6 +24,7 @@
 -include Custom.make
 SUBDIRS = $(_CUSTOM_SUBDIRS_) \
 	asn1 \
+	ceph \
 	docsis \
 	ethercat \
 	gryphon \
Index: epan/Makefile.am
===================================================================
--- epan/Makefile.am	(revision 45655)
+++ epan/Makefile.am	(working copy)
@@ -209,6 +209,7 @@
 -include ../plugins/Custom.make
 plugin_src = \
 	../plugins/asn1/packet-asn1.c \
+	../plugins/ceph/packet-ceph.c \
 	../plugins/docsis/packet-bintrngreq.c \
 	../plugins/docsis/packet-bpkmattr.c \
 	../plugins/docsis/packet-bpkmreq.c \
Index: packaging/nsis/Makefile.nmake
===================================================================
--- packaging/nsis/Makefile.nmake	(revision 45655)
+++ packaging/nsis/Makefile.nmake	(working copy)
@@ -46,6 +46,7 @@
 
 PLUGINS= \
 	../../plugins/asn1/asn1.dll \
+	../../plugins/ceph/ceph.dll \
 	../../plugins/docsis/docsis.dll \
 	../../plugins/ethercat/ethercat.dll \
 	../../plugins/gryphon/gryphon.dll \
Index: packaging/nsis/wireshark.nsi
===================================================================
--- packaging/nsis/wireshark.nsi	(revision 45655)
+++ packaging/nsis/wireshark.nsi	(working copy)
@@ -852,6 +852,7 @@
 ;-------------------------------------------
 SetOutPath '$INSTDIR\plugins\${VERSION}'
 File "${STAGING_DIR}\plugins\${VERSION}\asn1.dll"
+File "${STAGING_DIR}\plugins\${VERSION}\ceph.dll"
 File "${STAGING_DIR}\plugins\${VERSION}\docsis.dll"
 File "${STAGING_DIR}\plugins\${VERSION}\ethercat.dll"
 File "${STAGING_DIR}\plugins\${VERSION}\gryphon.dll"
